outputs.tf

```diff
@@ -17,3 +17,8 @@ output "this_dynamodb_table_stream_label" {
   description = "A timestamp, in ISO 8601 format of the Table Stream. Only available when var.stream_enabled is true"
   value       = var.stream_enabled ? concat(aws_dynamodb_table.this.*.stream_label, [""])[0] : null
 }
+
+output "this_dynamodb_table_name" {
+  description = "Name of the DynamoDB table"
+  value       = var.name
```

Comment:
You should get `name` from the resource, if possible, like this:
```hcl
value = element(concat(aws_dynamodb_table.this.*.name, [""]), 0)
```

Reply:
Thank you for the prompt response
I am getting error for this code
```
output "dynamodb_name" {
//  value = element(concat(aws_dynamodb_table.this.*.name, [""]), 0)
  value = element(concat(module.dynamodb_table.this.*.name, [""]), 0)
}
```
Not sure how to fix this
```
Error: Unsupported attribute

  on outputs.tf line 27, in output "dynamodb_name":
  27:   value = element(concat(module.dynamodb_table.this.*.name, [""]), 0)

This object does not have an attribute named "this".


```

Reply:
It should be from `aws_dynamodb_table.this` and not from `module.dynamodb_table.this`